sensor_msgs/Image im
---
float64 angle